packages/browser/src/errors.ts

```diff
@@ -5,6 +5,10 @@ const logtoClientErrorCodes = Object.freeze({
   sign_in_session: {
     invalid: 'Invalid sign-in session.',
   },
+  not_authenticated: 'Not authenticated.',
+  get_access_token_by_refresh_token_failed: 'Failed to get access token by refresh token.',
+  fetch_user_info_failed: 'Unable to fetch user info. Invalid access token.',
+  invalid_id_token: 'Invalid id token.',
```

Comment:
it'll be better to use scoped keys here, say `{ get_access_token: { not_authenticated: 'Not ...', ... } }`